README.md

```diff
@@ -75,7 +75,7 @@ Parameters are provided to configure the behavior of the bridge. These parameter
  * __param_whitelist__: List of regular expressions ([ECMAScript grammar](https://en.cppreference.com/w/cpp/regex/ecmascript)) of whitelisted parameter names. Defaults to `[".*"]`.
   * __client_topic_whitelist__: List of regular expressions ([ECMAScript grammar](https://en.cppreference.com/w/cpp/regex/ecmascript)) of whitelisted client-published topic names. Defaults to `[".*"]`.
  * __send_buffer_limit__: Connection send buffer limit in bytes. Messages will be dropped when a connection's send buffer reaches this limit to avoid a queue of outdated messages building up. Defaults to `10000000` (10 MB).
- * __use_compression__: Use websocket compression (permessage-deflate). Suited for connections with smaller bandwith, at the cost of additional CPU load.
+ * __use_compression__: Use websocket compression (permessage-deflate). It is recommended to leave this turned off as messages of subscribed topics are transferred in binary form for which compression is ineffective. Defaults to `false`.
```

Comment:
Compression effectiveness isn't related to whether data is binary or ASCII/UTF-8, there has to be a better way to communicate why per-message deflate compression is generally not effective for the type of data that is commonly streamed over this websocket protocol.

Reply:
```suggestion
 * __use_compression__: Use websocket compression (permessage-deflate). It is recommended to leave this turned off as it increases CPU usage and per-message compression often yields low compression ratios for robotics data. Defaults to `false`.
```

Reply:
much better, thanks!